html5/index.html

```diff
@@ -23,47 +23,14 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="stylesheet" href="css/spinner.css" />
 
-    <script type="text/javascript" src="js/lib/jquery.js"></script>
-    <script type="text/javascript" src="js/lib/jquery-ui.js"></script>
-    <script
-      type="text/javascript"
-      src="js/lib/jquery.ba-throttle-debounce.js"
-    ></script>
-    <script
-      type="text/javascript"
-      src="js/lib/jquery-transform-draggable.js"
-    ></script>
-
-    <script type="text/javascript" src="js/lib/rencode.js"></script>
-    <script type="text/javascript" src="js/lib/bencode.js"></script>
-    <script type="text/javascript" src="js/lib/zlib.js"></script>
-    <script type="text/javascript" src="js/lib/lz4.js"></script>
-    <script type="text/javascript" src="js/lib/brotli_decode.js"></script>
-    <script type="text/javascript" src="js/lib/forge.js"></script>
-
-    <script type="text/javascript" src="js/lib/jsmpeg.js"></script>
-    <script type="text/javascript" src="js/lib/broadway/Decoder.js"></script>
-    <script type="text/javascript" src="js/lib/aurora/aurora.js"></script>
-    <script type="text/javascript" src="js/lib/aurora/aurora-xpra.js"></script>
-
-    <script type="text/javascript" src="js/lib/StreamSaver.js"></script>
-    <script type="text/javascript" src="js/lib/FileSaver.js"></script>
-    <script type="text/javascript" src="js/lib/jszip.js"></script>
-    <script type="text/javascript" src="js/lib/detect-zoom.js"></script>
```

Comment:
Which of these unimported `lib`s (`jsmpeg`, `aurora`, `detect-zoom`, etc.) are unused/unwanted vs. still-used/wanted for their side effects?

Reply:
We need all of those!
Why would you remove them?

File transfers and stream saving are essential: #120
`jszip` is for zlib, which I intend to remove completely.
`detect-zoom` is used in `Window.js`.

Reply:
I moved most of those to import statements at the top of the relevant source files. However, for some of them like `detect-zoom` I didn't see any variable-level dependencies that warranted any import statements. However, if there are implicit dependencies like `Window.js` on `detect-zoom` then I'll definitely add those! I can't yet test this branch to find those implicit dependencies at runtime. Do the import moves make sense?

Reply:
Oh, now I see. Much nicer!